docs/cody/proxy-setup.mdx

```diff
@@ -0,0 +1,112 @@
+
+# Proxy Setup Guide
+
+## Introduction
+This guide will walk you through setting up HTTP, HTTPS, and SOCKS proxies in Visual Studio Code (VS Code) and JetBrains IDEs. It also includes instructions for handling self-signed certificates on macOS and Windows.
+
+## Proxy Setup for VS Code
+
+### Accessing the Proxy Settings
```

Comment:
Let's add this disclaimer that not setting up proxy may surface like this. @MaedahBatool please advise on the placement of this callout after we figured out the tabs